README.md

```diff
@@ -142,8 +142,13 @@ debug('this is hex: %h', new Buffer('hello world'))
 ```
 
 ## Browser support
+  You can build a browser-ready script using [browserify](https://github.com/substack/node-browserify),
```

Comment:
I'm not familiar with webpack or anything besides browserify. Please add some info if you are able to. Otherwise, I still consider this PR an improvement as-is.